Content.Shared/AirlockPainter/Prototypes/AirlockGroupPrototype.cs

```diff
@@ -0,0 +1,21 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.AirlockPainter.Prototypes
+{
+    [Prototype("AirlockGroup")]
+    public sealed class AirlockGroupPrototype : IPrototype
+    {
+        [ViewVariables]
+        [DataField("id", required: true)]
```

Comment:
Some changes were recently made to how prototypes work, and now the mandatory ID field has its own attribute that it needs to use.
```suggestion
        [IdDataField]
```